### PR TITLE
Sftp ftp merge

### PR DESCRIPTION
**Pull request by adolfoarmas**
Fixes https://github.com/parkpow/deep-license-plate-recognition/issues/97

### CODE REVIEW

File: ftp_and_sftp_processor.py [hunk 1]
```diff
@@ -0,0 +1,631 @@
...
+        pass
+
+    @abstractmethod
+    def retrieve_files(self):
+        pass
+
+    def processing_single_camera(self, args):
+        file_list = []
+        dirs = []
+        nondirs = []
```
Comment (stevelukis) [inline ftp_and_sftp_processor.py:70]: We can remove this empty array initialization as they aren't needed

File: ftp_and_sftp_processor.py [hunk 2]
```diff
@@ -0,0 +1,631 @@
...
+import os
+import stat
+import sys
+import tempfile
+import time
+import ftplib
+from abc import ABC, abstractmethod
+from datetime import datetime, timedelta
+from ftplib import FTP, error_perm, error_reply
+from pathlib import Path
```
Comment (stevelukis) [inline ftp_and_sftp_processor.py:14]:
```
import stat
...
from pathlib import Path
```

These imports are unused

File: ftp_and_sftp_processor.py [hunk 3]
```diff
@@ -0,0 +1,631 @@
...
+                        ]
+                    )
+
+
+class SFTPProcessor(FileTransferProcessor):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.sftp = None
+        self.os_linux = True
+        #print(self.__dict__)
```
Comment (stevelukis) [inline ftp_and_sftp_processor.py:342]: let's remove this debug code

**Review by stevelukis — COMMENTED** [review]
@adolfoarmas Looks good overall, just need some minor clean up

**Review by stevelukis — APPROVED** [review]

**Comment by stevelukis** [comment]
@marcbelmont looks good to me

**Comment by adolfoarmas** [comment]
@stevelukis my bad, i did ask you again to review by mistake

**Review by stevelukis — APPROVED** [review]

**Comment by marcbelmont** [comment]
@adolfoarmas is it now safe to remove the old files: [sftp_processor.py](https://github.com/parkpow/deep-license-plate-recognition/blob/master/sftp_processor.py) and [ftp_processor.py](https://github.com/parkpow/deep-license-plate-recognition/blob/master/ftp_processor.py) ?

I believe we should update this [README.md](https://github.com/parkpow/deep-license-plate-recognition/blob/master/README.md)

**Pull request merged**